Description: change paths so examples build within Debian examples directory.
Author: Alastair McKinstry <mckinstry@debian.org>
Uploaded: no

Index: lcov-1.12/example/Makefile
===================================================================
--- lcov-1.12.orig/example/Makefile
+++ lcov-1.12/example/Makefile
@@ -10,10 +10,10 @@
 CC      := gcc
 CFLAGS  := -Wall -I. -fprofile-arcs -ftest-coverage
 
-LCOV    := ../bin/lcov
-GENHTML := ../bin/genhtml
-GENDESC := ../bin/gendesc
-GENPNG  := ../bin/genpng
+LCOV    := /usr/bin/lcov
+GENHTML := /usr/bin/genhtml
+GENDESC := /usr/bin/gendesc
+GENPNG  := /usr/bin/genpng
 
 # Depending on the presence of the GD.pm perl module, we can use the
 # special option '--frames' for genhtml
